### ISSUE TITLE
Add file locator shortcut and popup

### ISSUE THREAD

**Issue by adamfowleruk** [HaikuArchives/Paladin#268]
E.g. Ctrl+K highlights a text entry area, typing Proje would list Project.h Project.cpp ProjectTests.cpp to open. Hit enter to locate in the Project window and open in the preferred editor.
From @diversys

**Comment by adamfowleruk** [comment]
Extension: Support typing 'qfw' to return 'QuickFindWindow'

**Comment by adamfowleruk** [comment]
@diversys I've now added Quick Find / Locator to Paladin 2.2. Added to haikuports and waiting for it to be merged in. A few bug fixes in there too.
I'm leaving this issue open as I want to enhance the feature. Please provide feedback on its usability. I'm aware it requires keyboard arrows up/down and enter rather than mouse right now. It also requires exact capitalisation at the moment within file names.

**Comment by diversys** [comment]
I'll try to check it out soon. Please also see enhancement video here https://www.youtube.com/watch?v=JELdkAuuPRI

**Comment by adamfowleruk** [comment]
I've now added shortcut support. E.g. qfw matches QuickFindWindow.cpp

**Comment by adamfowleruk** [comment]
@diversys I've concentrated on matching file names as this was in an earlier video. In the one you posted he appears to be searching for matches of file content too. I could change the feature so if the search text starts with a " character it will search text content instead of file name. Is this desirable?

**Comment by diversys** [comment]
Sure. BTW, what about creating a locator widget (like he did) instead of opening separate window?

**Comment by adamfowleruk** [comment]
Yeah I've not figured out how to do that yet! 8o) Ideally I want a popup frame without a window name or controls that should vanish when the focus leaves the window, much like in visual studio code and its search widget.

**Comment by adamfowleruk** [comment]
Moving the full text search feature to advanced search #302 as it will require advanced indexing.

**Comment by adamfowleruk** [comment]
@diversys FYI I've made the quick find act like a thin popup now. If you focus outside the window or hit escape, it vanishes. If you use key up/down and hit enter it opens a file and closes, same for double click. Now a much nicer experience. Latest develop branch has the improvements. Still appears in the centre of the screen because I couldn't think of a better location! :D

**Comment by diversys** [comment]
Paladin from develop branch doesn't build here, it compiles after i fix MonitorWindow.h path
```
diff --git a/Paladin/ProjectWindow.cpp b/Paladin/ProjectWindow.cpp
index d8f75d0..39bb0c9 100644
--- a/Paladin/ProjectWindow.cpp
+++ b/Paladin/ProjectWindow.cpp
@@ -59,7 +59,7 @@
 #include "LibWindow.h"
 #include "LicenseManager.h"
 #include "Makemake.h"
-#include "MonitorWindow.h"
+#include "PreviewFeatures/MonitorWindow.h"
```
 but it then fails to link https://pastebin.com/raw/DhrMEFp5

**Comment by adamfowleruk** [comment]
Fixed by #305.
